src/components/Footer/Footer.js

```diff
@@ -7,29 +7,44 @@ import Copyright from 'react-spatial/components/Copyright';
 import Select from '@geops/react-ui/components/Select';
 import MousePosition from 'react-spatial/components/MousePosition';
 import ActionLink from '@geops/react-ui/components/ActionLink';
+import coordinateHelper from '../../utils/coordinateHelper';
 import {
   setLanguage,
   setProjection,
   setDialogVisible,
 } from '../../model/app/actions';
 import './Footer.scss';
 
-const numberFormat = coords => {
-  const coordStr = coords.map(num =>
-    Math.round(num)
-      .toString()
-      .replace(/\B(?=(\d{3})+(?!\d))/g, "'"),
-  );
-
-  return coordStr;
-};
-
 const Footer = () => {
   const dispatch = useDispatch();
   const { t } = useTranslation();
   const layerService = useSelector(state => state.app.layerService);
   const map = useSelector(state => state.app.map);
   const language = useSelector(state => state.app.language);
+  const projection = useSelector(state => state.app.projection);
+
+  const projections = [
+    {
+      label: 'CH1093 / LV03',
+      value: 'EPSG:21781',
+      format: c => `${t('Koordinaten')}: ${coordinateHelper.meterFormat(c)}`,
+    },
+    {
+      label: 'CH1093+ / LV95',
+      value: 'EPSG:2056',
+      format: c => `${t('Koordinaten')}: ${coordinateHelper.meterFormat(c)}`,
+    },
+    {
+      label: 'Web Mercator',
+      value: 'EPSG:3857',
+      format: c => `${t('Koordinaten')}: ${coordinateHelper.meterFormat(c)}`,
+    },
+    {
+      label: 'WGS 84',
+      value: 'EPSG:4326',
+      format: c => `${t('Koordinaten')}: ${coordinateHelper.wgs84Format(c)}`,
+    },
+  ];
```

Comment:
I would  put the projections array outside the functionnal component  so we create it only once.

Reply:
@oterral But we need the translation function

Reply:
yes I hadn't seen